jwt-auth/src/main/java/fish/payara/microprofile/jwtauth/cdi/CdiExtension.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 2017 Payara Foundation and/or its affiliates. All rights reserved.
```

Comment:
Was this class moved or is the year a copy-paste error?

Reply:
All of the classes moved from https://github.com/payara/payara/ repository.